portfolio/src/main/java/com/google/sps/servlets/ChartsServlet.java

```diff
@@ -0,0 +1,46 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import java.util.ArrayList;
+import java.util.Scanner;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/charts")
+public class ChartsServlet extends HttpServlet {
+
+  private LinkedHashMap<String, ArrayList<Integer>> genderPopularity = new LinkedHashMap<>();
+
+  @Override
+  public void init() {
+    Scanner scanner = new Scanner(getServletContext().getResourceAsStream(
+        "/WEB-INF/anime-movie-popularity-by-gender.csv"));
+
+    scanner.nextLine();  // First line is column titles.
+    while (scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] data = line.split(",");
+
+      String favorability = data[0];
```

Comment:
am i confused or is this supposed to be the gender?

Reply:
Oh right, I forgot to change the variable names after changing the CSV files. Thank you